homeassistant/components/homematicip_cloud.py

```diff
@@ -106,13 +108,68 @@ def __init__(self, hass, config, websession):
         self.home = AsyncHome(hass.loop, websession)
         self.home.set_auth_token(_authtoken)
 
+        self.home.on_update(self.update)
+        self._accesspoint_connected = True
+
         hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, self.close())
 
     async def init(self):
         """Initialize connection."""
         await self.home.init(self._accesspoint)
         await self.home.get_current_state()
 
+    def update(self, *args, **kwargs):
```

Comment:
Is this called from within the event loop? If so, please mark it as async (via @callback if it's not doing awaits), and add `async_` to the name and the doc string.

Reply:
`update` is called inside the event loop. Updated according to your suggestions.